content/blog/2020-07-03-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Remote Optimization Improvements in DVC 1.0
+date: 2020-07-03
+description: |
+  An overview of how data synchronization to and from remote storage is optimized in DVC 1.0.
+picture: ''
+author: peter_rowlands
+---
+
+At its core, [DVC](/) is simply a data management tool. While DVC does include
+many additional [features](/features) for machine learning (ML) workflows, DVC
+can be used track and version files and directories in any context, even in
+scenarios involving large amounts of data (both in terms of file size and file
+count).
+
+One of the key data management features provided by DVC is the ability to
+efficiently synchronize versioned datasets between a user's local machine and
+(platform agnostic) [cloud (remote) storage](/doc/command-reference/remote). Our
+users have frequently stressed the need for us to optimize these synchronization
+operations wherever possible, and one of our key goals while developing DVC 1.0
+was to improve performance in this area.
+
+![](/uploads/images/2020-07-03/optimization_screenshot.png '=600')
+
+In particular, we identified one bottleneck which affects all cloud sync tools -
+remote status queries. Any data sync tool will be restricted by certain limits
+imposed by cloud storage APIs - namely, the actual API methods available for
+querying file status. Using one method over another could mean the difference
+between a given sync operation taking seconds rather than hours to complete.
+
+And with the release of version 1.0, DVC now includes several performance
+enhancements to address this issue, ensuring that we always optimize requests
+made to cloud storage APIs.
+
+_Note: "Cloud storage" and "remote storage" will be used interchangeably
+throughout this post. When discussing dataset size in this post, we mean size in
+terms of total number of files in a dataset, rather than the total amount of
+file data (bytes)._
+
+## Why status queries are a performance bottleneck
+
+Using cloud storage will require some solution for synchronizing data between
```

Comment:
It still feels to me that this can become an intro to the whole blog post. You can just remove with some adjustments pretty much everything from the second paragraph of the intro up to this point. Or may be even start here and put a general small paragraph about DVC a bit later.